bbb-common-web/src/main/scala/org/bigbluebutton/api2/util/RecordingMetaXmlHelper.scala

```diff
@@ -188,19 +190,35 @@ class RecMetaXmlHelper extends RecordingServiceGW with LogHelper {
     }
   }
 
-  def getRecordingTextTracks(recordId: String, captionsDir: String): String = {
+  def getRecordingTextTracks(recordId: String, captionsDir: String, captionBasUrl: String): String = {

```

Comment:
You have this typo in several places: `captionBasUrl` should be `captionBaseUrl`